### PR TITLE
Update custom WDA server initialization script

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
## Proposed changes

Update the example script to reflect the recent changes in the integration with WDA

### CODE REVIEW

**Review by imurchie — APPROVED** [review]
Looks good.

File: docs/en/advanced-concepts/wda-custom-server.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ cd /usr/local/lib/node_modules/appium/node_modules/appium-xcuitest-driver/WebDri
 ./Scripts/bootstrap.sh -d
 ```
 
+Also, it might be necessary to create an empty folder for WDA resources:
+
+```bash
+mkdir -p /usr/local/lib/node_modules/appium/node_modules/appium-xcuitest-driver/WebDriverAgent/Resources/WebDriverAgent.bundle
```
Comment (jlipps) [inline docs/en/advanced-concepts/wda-custom-server.md]: why don't we just do this in appium rather than make the user do it?

**Review by jlipps — APPROVED** [review]

File: docs/en/advanced-concepts/wda-custom-server.md (hunk 1 above)
Comment (imurchie) [inline docs/en/advanced-concepts/wda-custom-server.md]: We do, if they're allowing Appium to automate the process. But we can't if they are managing things themselves.

Comment (jlipps) [inline docs/en/advanced-concepts/wda-custom-server.md]: Right, these are the instructions for rolling your own. 👍

**Comment by messianic** [comment]
What about the import statements?? Couldn't find the time delta class

**Pull request merged**